src/pybind/mgr/dashboard/services/rgw_client.py

```diff
@@ -448,14 +448,17 @@ def bucket_exists(self, bucket_name, userid, request=None):
             raise e
 
     @RestClient.api_put('/{bucket_name}')
-    def create_bucket(self, bucket_name, zonegroup, placement_target, request=None):
+    def create_bucket(self, bucket_name, zonegroup=None, placement_target=None, request=None):
         logger.info("Creating bucket: %s, zonegroup: %s, placement_target: %s",
                     bucket_name, zonegroup, placement_target)
-        create_bucket_configuration = ET.Element('CreateBucketConfiguration')
-        location_constraint = ET.SubElement(create_bucket_configuration, 'LocationConstraint')
-        location_constraint.text = '{}:{}'.format(zonegroup, placement_target)
-
-        return request(data=ET.tostring(create_bucket_configuration, encoding='utf-8'))
+        data = None
+        if zonegroup is not None and placement_target is not None:
```

Comment:
maybe it can be rewritten as: `if zonegroup and placement_target:`?

Reply:
Done.